### DISCUSSION TITLE
Repeater subfield handles

### DISCUSSION THREAD

**Discussion by jmauzyk** [verbb/formie#1533] [Q&A]
Not sure if this is more of a bug or feature request, so starting as a discussion here. If for example I create a field with the label `Email` and then create repeater subfield with the same name, they both end up with the handle `email`.

On a multi-page form, this results in new repeater rows having their Email fields pre-populated with the other field's value if it was filled on a previous page.

I'm wondering if this is expected behavior, or if this is a bug. If it's expected behavior, would it be possible to update the form editor so that newly-added repeater subfields automatically update duplicate handles to avoid conflicts?

**Comment by engram-design** [comment]
That's indeed a bug, opening as an issue
